src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -217,6 +217,8 @@ std::vector<std::string> disabledTestPatterns() {
     retVector.emplace_back(R"(smoke_NegativeQuantizedMatMulMultiplyFusion.*)");
     // int8 specific
     retVector.emplace_back(R"(smoke_Quantized.*)");
+    //ACL does not support FakeQuantize fusing for MatMul
+    retVector.emplace_back(R"(.*_(FC|MM)_.*Fused=.*FakeQuantize.*)");
```

Comment:
Why don't we make fusing instances to be x64 specific?
I guess ACL won't support this fusing in the nearest future.

Reply:
addressed